templates/cluster-network-addons/VERSION/operator.yaml.in

```diff
@@ -36,3 +36,61 @@ subjects:
     name: cluster-network-addons-operator
 
 {{.CNA.Deployment}}
+
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cnao-allow-egress-to-dns
+  namespace: {{.Namespace}}
+spec:
+  podSelector:
+    matchLabels:
+      name: cluster-network-addons-operator
+  policyTypes:
+    - Egress
+  egress:
+    - to:
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: kube-system
```

Comment:
this is problematic, it wont be dual right ?

(this is partial review, but it seems the PR is not ready yet as it is now if so)